README.md

```diff
@@ -22,7 +22,7 @@ How to make your own entries:
 
 Winapp2 entries follow ABC ordering and we currently support all of the following code:
 
-[Name *] Name of the entry. Please include the space between the name and the *.
+[Name \*] Name of the entry. Please include the space between the name and the \*.
```

Comment:
This needs to be worded better, otherwise new people might think they need to include the \.